packages/components/src/highlight-cards/weekly-highlight-cards.tsx

```diff
@@ -64,49 +73,65 @@ export default function WeeklyHighlightCards( {
 	return (
 		<div className={ classNames( 'highlight-cards', className ?? null ) }>
 			<h3 className="highlight-cards-heading">
-				<span>{ translate( '7-day highlights' ) }</span>
-				<div className="highlight-cards-heading__tooltip">
-					<span
-						className="highlight-cards-heading-icon"
-						ref={ textRef }
-						onMouseEnter={ () => setTooltipVisible( true ) }
-						onMouseLeave={ () => setTooltipVisible( false ) }
-					>
-						<Icon className="gridicon" icon={ info } />
-					</span>
-					<Popover
-						className="tooltip tooltip--darker highlight-card-tooltip"
-						isVisible={ isTooltipVisible }
-						position="bottom right"
-						context={ textRef.current }
-					>
-						<div className="highlight-card-tooltip-content comparing-info">
-							<p>
-								{ translate( 'Highlights displayed are for the last 7 days, excluding today.' ) }
-							</p>
-							<p>
-								{ translate(
-									'Trends shown are in comparison to the previous 7 days before that.'
-								) }
-							</p>
-							<div className="comparing-info-chart">
-								<small>
-									{ translate( '%(fourteen)d days {{vs/}} %(seven)d days', {
-										components: {
-											vs: <span>vs</span>,
-										},
-										args: {
-											fourteen: 14,
-											seven: 7,
-										},
-									} ) }
-								</small>
-								{ comparingInfoRangeChart }
-								{ comparingInfoBarsChart }
+				<span>
+					{ currentPeriod === PAST_THIRTY_DAYS
+						? translate( '30-day highlights' )
+						: translate( '7-day highlights' ) }
+				</span>
+
+				{ isHighlightsSettingsEnabled && (
+					<small className="highlight-cards-heading__description">
+						{ currentPeriod === PAST_THIRTY_DAYS
+							? translate( 'Compared to previous 30 days' )
+							: translate( 'Compared to previous 7 days' ) }
+					</small>
+				) }
+
+				{ ! isHighlightsSettingsEnabled && (
```

Comment:
Just a note: The tooltip is hidden when settings is enabled. I guess this is intentional.

Reply:
Yes, it is according to the design.

<img width="962" alt="241165976-1f1a76c3-150b-4e15-9096-e7fce5a2f99f" src="https://github.com/Automattic/wp-calypso/assets/6869813/fa793323-9b41-49df-bc61-ef42c2e96f33">